### ISSUE TITLE
Building with nightly problem 

### ISSUE THREAD

**Issue by gbip** [Manishearth/rust-clippy#1604]
First of all, here are my cargo and rustc version :

`cargo 0.18.0-nightly (5db6d64 2017-03-03)`
`rustc 1.17.0-nightly (b1e31766d 2017-03-03)`

I added clippy to my Cargo.toml a few weeks ago, and now, it doesn't seem to compile anymore.
Here are the parts related to clippy in the Cargo.toml :
```
[dependencies]
clippy = {version = "0.0.*", optional = true}

[features]
default = ["clippy"]
```
When running `cargo build` the build fail throwing the following error :
```
   Compiling clippy_lints v0.0.117
error[E0308]: mismatched types
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/clippy_lints-0.0.117/src/cyclomatic_complexity.rs:49:36
   |
49 |         let cfg = CFG::new(cx.tcx, expr);
   |                                    ^^^^ expected struct `rustc::hir::Body`, found struct `rustc::hir::Expr`
   |
   = note: expected type `&rustc::hir::Body`
              found type `&'tcx rustc::hir::Expr`

error: aborting due to previous error

error: Could not compile `clippy_lints`.

```

Can you fix this issue ?
It seems like there is a v0.0.118 that is on the way and that seems to be compiling to if you could just push it to crates.io.
Thanks,

**Comment by oli-obk** [comment]
It's already fixed, I was just waiting for travis:  #1603

**Comment by oli-obk** [comment]
and the new version is published
